File: src/main/kotlin/org/kite9/diagram/dom/painter/SVGContainerRectangularPainter.java
package org.kite9.diagram.dom.painter;

import org.kite9.diagram.dom.bridge.ElementContext;
import org.kite9.diagram.model.style.DiagramElementType;
import org.w3c.dom.Element;

/**
 * Handles painting for {@link DiagramElementType.CONTAINER}
 * 
 * This implementation allows Containers to contain some SVG, but it cannot be used for sizing purposes.
 * 
 * @author robmoffat
 *
 */
public class SVGContainerRectangularPainter extends DirectSVGGroupPainter {

	public SVGContainerRectangularPainter(Element theElement, ElementContext ctx) {
		super(theElement);
	}

}

File: visualization/kite9-visualization-common/src/visMain/kotlin/org/kite9/diagram/dom/painter/SVGContainerRectangularPainter.kt
package org.kite9.diagram.dom.painter

import org.kite9.diagram.dom.bridge.ElementContext
import org.kite9.diagram.dom.painter.DirectSVGGroupPainter
import org.w3c.dom.Element

/**
 * Handles painting for [DiagramElementType.CONTAINER]
 *
 * This implementation allows Containers to contain some SVG, but it cannot be used for sizing purposes.
 *
 * @author robmoffat
 */
class SVGContainerRectangularPainter(theElement: Element, ctx: ElementContext) : DirectSVGGroupPainter(theElement)